lib/decoding/index.js

```diff
@@ -72,7 +76,7 @@ const bufferToMeasurements = function bufferToMeasurements (buffer, bufferTransf
     }
   }
 
-  return result;
+  return [result, warnings];
```

Comment:
why not `{ result, warnings }`? It's more explicit and same amount of chars ;)

Reply:
Also changed with the last commit